Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 #!groovy
-@Library(['github.com/cloudogu/ces-build-lib@2.2.1', 'github.com/cloudogu/dogu-build-lib@v2.3.1']) _
+@Library(['github.com/cloudogu/ces-build-lib@2.3.0', 'github.com/cloudogu/dogu-build-lib@v2.3.1']) _
 import com.cloudogu.ces.cesbuildlib.*
 import com.cloudogu.ces.dogubuildlib.*
 import com.cloudogu.ces.zaleniumbuildlib.*
```

Comment:
Do we really need zaleniumbuildlib here?

Reply:
No, this statement doesn't seem to make sense as the library is not imported at the top. I'll delete it.